src/deploy_inst.h

```diff
@@ -81,6 +82,29 @@ class DeployInst : public cyclus::Institution {
     "uilabel": "Lifetimes" \
   }
   std::vector<int> lifetimes;
+
+ private:
+/////////// position toolkit ///////////
```

Comment:
This commented heading may be incompatible with our [style guide](http://fuelcycle.org/arche/index.html#a-word-about-style).

Reply:
Apologies for the inconsistency.
I'll fix this right away!